CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * WARNING: The CLI now uses POSIX-style (double-dash `--flag`) for long-form flags. It will temporarily accept the single-dash `-flag` form with a warning, which will become an error in a future release (https://github.com/sigstore/cosign/pull/835)
 * Added `sget` as part of Cosign's releases (https://github.com/sigstore/cosign/pull/752)
 * The `copasetic` binary was unceremoniously [baleeted](https://www.youtube.com/watch?v=07h0ksKx5sM) (https://github.com/sigstore/cosign/pull/785)
+* Release images are now shipped for every architecture distroless ships on (currently: amd64, arm64, arm, s390x, ppc64le) (https://github.com/sigstore/cosign/pull/973)
```

Comment:
I think my release note was wrong.  The CI images are, but the release images go through another path.

Reply:
updated and moved it down to enhancements